index.html

```diff
@@ -68,6 +67,123 @@ <h2 id="support-text">I'm a Software Developer</h2>
     <img id="main-img" src="assets/illustration.svg" alt="">
   </section>
 
+  <section id="recent-works">
+    <div class="container">
+      <div class="section-title">
+        <h1 id="recent-works-title">My Recent Works</h1>
```

Comment:
- [ ] There are about six projects on the Figma design but just one on your design, please kindly add more projects.

Reply:
I added more sections to the project to amount a total of 6